packages/apps/plugins/plugin-navtree/src/NavTreePlugin.tsx

```diff
@@ -70,9 +80,9 @@ export const NavTreePlugin = (): PluginDefinition<NavTreePluginProvides> => {
       getActions(graph, state.root);
 
       // TODO(thure): Do this dynamically.
-      state.openItemIds = { root: true, 'dxos.org/plugin/space-spaces': true, 'dxos.org/plugin/files': true };
+      settings.prop({ key: 'openItemIds', storageKey: 'openItemIds', type: LocalStorageStore.json() });
```

Comment:
```suggestion
      settings.prop({ key: 'openItemIds', storageKey: 'open-item-ids', type: LocalStorageStore.json() });
```
established pattern is to use kebab case for the storage key

Reply:
I found instances of both patterns established — this probably shouldn’t be a dev’s responsibility to figure out, given naming is among the hardest of coding problems.

Reply:
yeah, this is a temporary crutch. At some point this will be done automatically. The expectation is that `create` will have pluggable persistence (e.g. echo, local storage, etc.)